### PR TITLE
#6 Hero Section 

### PR DESCRIPTION
**Pull request by Sarthak0006**
I added navbar with hero section please merge my PR.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,117 +1,154 @@
...
+      <h3>Whether you're a beginner, expert, developer, or</h3>
+      <h3>designer, everyone is welcome to join our</h3>
+      <h3>international network of users.</h3>
+      <button id="contact-sales">JOIN OUR COMMUNITY</button>
+    </div>
+    <div class="header-image">
+      <img src="assets/images/Illustrations.svg" alt="Image">
     </div>
+  </div>
+  <!--End of Hero section-->
```
Comment (phyleria) [inline index.html:68]: Hi @Sarthak0006 please submit changes for only NavBar and Hero section. 
The other changes interfere with the other sections.

File: assets/css/style.css [hunk 2]
```diff
@@ -4,302 +4,339 @@ body {
...
+   color: #333;
+   text-decoration: none;
+   padding-left: 1vw;
+   padding-right: 1vw;
+}
+
+.nav-link ul li a:hover {
+   color: #670B78;
+}
+
```
Comment (phyleria) [inline assets/css/style.css:41]: Kindly add
 .nav-item{
   margin-right: 5%;
}
here

Comment (Sarthak0006) [reply assets/css/style.css:41]: Ok

Comment (phyleria) [reply assets/css/style.css:41]: Oh, 
I noticed you had accidentally omitted some code. Kindly push the first changes and we'll go through the rest of the code

**Review by phyleria — APPROVED** [review]
I'll go ahead and merge this.
Awesome work @Sarthak0006 !

**Comment by Sarthak0006** [comment]
> I'll go ahead and merge this. Awesome work @Sarthak0006 !

Thanks

**Pull request merged**